Doc/tools/static/rtd_switcher.js

```diff
@@ -0,0 +1,82 @@
+ function onSwitch(event) {
+     const option = event.target.selectedIndex;
+     const item = event.target.options[option];
+     window.location.href = item.dataset.url;
+ }
+
+ document.addEventListener("readthedocs-addons-data-ready", function(event) {
+   const config = event.detail.data()
+
+   // Add some mocked hardcoded versions pointing to the official
+   // documentation while migrating to Read the Docs.
+   // These are only for testing purposes.
+   // TODO: remove them when managing all the versions on Read the Docs,
+   // since all the "active, built and not hidden" versions will be shown automatically.
+   let versions = config.versions.active.concat([
+       {
+           slug: "dev (3.13)",
+           urls: {
+               documentation: "https://docs.python.org/3.13/",
+           }
+       },
```

Comment:
See https://github.com/python/cpython/pull/119539 file changed, add 3.14

```suggestion
    {
           slug: "dev (3.14)",
           urls: {
               documentation: "https://docs.python.org/3.14/",
           }
       },
       {
           slug: "pre (3.13)",
           urls: {
               documentation: "https://docs.python.org/3.13/",
           }
       },
```

Reply:
Thanks, let's add it in https://github.com/python/cpython/pull/119539, so that this PR is only moving code without adding to it.